internal/sys/password_policy.go

```diff
@@ -21,7 +23,7 @@ type PasswordPolicyResource struct {
 }
 
 func (r *PasswordPolicyResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
-	resp.TypeName = req.ProviderTypeName + "_password_policy"
+	resp.TypeName = req.ProviderTypeName + "_password_policy_fw"
```

Comment:
Temporarily names suffixed with `_fw` to avoid duplicate resource names and have green CI 🟢 This can be renamed to simply `vault_password_policy` once we are ready to deprecate the SDKv2 resource with the same name